bs/src/tablecloth.mli

```diff
@@ -1527,41 +1527,345 @@ module Result : sig
     -> unit
 end
 
+(**
+  This module implements the `Option` type, which has a variant for 
+  valid values (`'Some`), and one for invalid values (`'None`).
+*)
+
 module Option : sig
+  (**
+    `type` is the type constructor for an `Option` type. You specify
+    the type of the `Some` variant.
+    
+    ### Example
+    
+    Here is how you would annotate an `Option` variable whose `Some`
+    variant is an integer:
+    
+    ```ocaml
+    let x: (int) Tablecloth.Option.t = Some 3
+    let y: (int) Tablecloth.Option.t = None
```

Comment:
I dont think we should encourage people to use `Tablecloth.Option.t`

Reply:
Changed.